testing/cypress/support/commands.js

```diff
@@ -11,4 +11,177 @@ Cypress.Commands.add('signin', (username, password) => {
 
   cy.get('[id="loginConfirm"]').click()
 
-});
\ No newline at end of file
+});
+
+Cypress.Commands.add('completeForm', () => {
```

Comment:
added this for the workflows file to complete the form and then tests will be completed